src/nvim/charset.c

```diff
@@ -734,8 +735,8 @@ int vim_strnsize(char_u *s, int len)
 /// @return Number of characters.
 #define RET_WIN_BUF_CHARTABSIZE(wp, buf, p, col) \
   if (*(p) == TAB && (!(wp)->w_p_list || lcs_tab1)) { \
-    int ts; \
-    ts = (buf)->b_p_ts; \
+    int64_t ts; \
+    ts = (int)(buf)->b_p_ts; \
```

Comment:
Right, sorry about that.. 
I like the first one.
`const int ts = (int)  (buf)->b_p_ts;`
Why `const` though?